stm/stm.ml

```diff
@@ -1231,9 +1231,22 @@ end = struct (* {{{ *)
 
   let get_prev_proof ~doc id =
     try
-      let did = fold_until back_tactic 1 id in
-      get_proof ~doc did
-    with Not_found -> None
+      let np = get_proof ~doc id in
+      match np with
+      | None -> None
+      | Some cp ->
+        let did = ref id in
+        let rv = ref np in
+        let _done = ref false in
```

Comment:
why `_done`? The variable is used, did you get a warning?

Reply:
You mean why the name `_done` and not `done`?  `done` is a keyword, it can't be used for a variable name, right?

Reply:
Indeed, but then find another name. @gares is right: pre-pending `_` is conventionally reserved to unused variables.

Reply:
Note: for a function called `match`, we used `match_`!

Reply:
Ah, I was unaware of the convention.